### PR TITLE
Terms of engagement

### PR DESCRIPTION
**Pull request by char-star**
- adding terms of engagement to search/on_search
- adding tags to catalog scheme in beckn core (updated the beckn core zip)

### CODE REVIEW

File: api/components/examples/mutual-funds/confirm/confirm-lumpsum.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        "message_id": "bb579fb8-cb82-4824-be12-fcbc405b6608",
+        "version": "0.0.1",
+        "ttl": "PT10M",
+        "bpp_id": "api.sellerapp.com",
+        "bpp_uri": "https://api.sellerapp.com/ondc",
+        "action": "confirm"
+    },
+    "message": {
+        "order": {
+            "type": "PURCHASE",
```
Comment (92shreyansh) [inline api/components/examples/mutual-funds/confirm/confirm-lumpsum.json]: We discussed to get this removed

Comment (char-star) [reply api/components/examples/mutual-funds/confirm/confirm-lumpsum.json]: done, order type is removed

File: api/components/attributes/mutual-funds/index.yaml [hunk 2]
```diff
@@ -500,13 +500,13 @@ select:
             description: Time associated for recurring orders
           schedule:
             _description:
-              required: optional
+              required: mandatory
```
Comment (char-star) [inline api/components/attributes/mutual-funds/index.yaml:503]: for SIP orders, in `on_select`, `item.time` attribute is provided. So, only if `item.time` is provided, then `schedule` and `frequency` is mandatory. If `item.time` is not provided, `schedule` and `frequency` shouldn’t be checked. But currently the atrribute check is happening and the following error is thrown

```
Error validating examples with attributes Error: Key path time.schedule.frequency does not exist in the example object at select
    at handleError (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:357:9)
    at checkKeysExistence (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:387:9)
    at handleIfObjectIsArray (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:404:9)
    at checkKeysExistence (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:395:7)
    at validateObject (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:351:3)
    at validateExamplesAttributes (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:456:23)
    at async getSwaggerYaml (/Users/satish/cybrilla/ONDC-FIS-Specifications/api/components/app.js:323:7)
    ```

**Comment by 92shreyansh** [comment]
few conflicts in the PR can we resolve

**Comment by char-star** [comment]
> few conflicts in the PR can we resolve

conflicts are resolved

**Pull request merged**